README.md

```diff
@@ -345,47 +346,397 @@ clay export domain.com/_lists > lists.clay
 clay export domain.com/2017/02/some-slug.html
 
 # export built-in 'New Page Templates' list (page uris will be unprefixed)
-clay export domnain.com/_lists/new-pages 
+clay export domnain.com/_lists/new-pages
+```
+
+## Compile
+
+```bash
+clay compile [--watch] [--minify] [--inlined] [--linked] [--plugins <space-separated list of postcss plugins>] [--globs <space-separated glob strings>] [--reporter <reporter>]
+```
+
+Compile assets based on standardized Clay conventions. Assets are compiled to a `public` folder at the root of your Clay install (the directory where you run the `clay compile` command), with scripts (including templates), styles (including fonts), and media output to the `js`, `css`, and `media` folders. You may run `clay compile` to compile _all_ assets, or run any of its subcommands (`media`, `fonts`, `styles`, `templates`, `scripts`) to compile a specific type of asset.
+
+Specifying `--watch` on `claycli compile` or any of its subcommands will compile assets once, then watch source files (and their dependencies) for changes. Specifying `--minify` (or setting `CLAYCLI_COMPILE_MINIFIED`) will run assets through minification and bundling if applicable. The `CLAYCLI_COMPILE_ASSET_HOST` and `CLAYCLI_COMPILE_ASSET_PATH` variables are used by the `styles` and `fonts` subcommands to generate links to media and font files in the compiled CSS.
+
+#### Arguments
+
+* `-w, --watch` enables watching of source files after compilation
+* `-m, --minify` enables minification and bundling of source files
+* `-i, --inlined` enables the generation of base64 inlined font CSS
+* `-l, --linked` enables the generation of linked font CSS
+* `-p, --plugins` allows running additional postcss plugins when compiling styles
+* `-g, --globs` allows compiling additional JavaScript to `public/js/_global.js`
```

Comment:
Just my two cents but `globs` seems weird, why not `global`?

Reply:
You're passing in globs, and I want to be explicit about that (the name of the compiled file is less important, as its inclusion is automatic rather than something devs need to configure)

Reply:
per our chat, I am gonna remove / move the "compiles to global.whatev" somewhere else so it's not confusing